conf/application.conf

```diff
@@ -86,10 +86,17 @@ auditing {
   }
 }
 
+# Suppress the auditing of assets as per PLATUI-837
 controllers {
   controllers.Assets = {
     needsAuditing = false
   }
+  uk.gov.hmrc.govukfrontend.controllers.Assets = {
+    needsAuditing = false
+  }
+  uk.gov.hmrc.hmrcfrontend.controllers.Assets = {
```

Comment:
Can we set these within the libraries?

Reply:
Good suggestion. However, I wonder if play-frontend is the right place as we would then be including knowledge of bootstrap play within the frontend libraries. [This](https://github.com/hmrc/bootstrap-play/blob/83e0aa63637d1fc9800e656c3f1d2a048e6411b4/bootstrap-frontend-play-26/src/main/resources/frontend.conf#L62) might be a good place though.

Reply:
Agree with @natcarey that it would be a good future piece of work to disable centrally, however in the interim this change ensures we're not flooding data lake with requests.